### PR TITLE
Code cleanups

### PR DESCRIPTION
**Pull request by qhy040404**

### CODE REVIEW

File: app/src/main/kotlin/com/absinthe/libchecker/ui/about/AbsAboutActivityProxy.java [hunk 1]
```diff
@@ -110,50 +107,44 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        givenInsetsToDecorView = true;
+      } else if (givenInsetsToDecorView) {
+        ViewCompat.onApplyWindowInsets(
+          decorView,
+          new WindowInsetsCompat.Builder()
+            .setInsets(
+              WindowInsetsCompat.Type.navigationBars(),
+              Insets.of(navigationBarsInsets.left, navigationBarsInsets.top, navigationBarsInsets.right, 0)
+            )
+            .build()
```
Comment (Goooler) [inline app/src/main/kotlin/com/absinthe/libchecker/ui/about/AbsAboutActivityProxy.java:141]: Rewrite all java to Kotlin? With better lambdas.

File: app/src/main/kotlin/com/absinthe/libchecker/ui/about/AbsAboutActivityProxy.java [hunk 2]
```diff
@@ -110,50 +107,44 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        );
```
Comment (Goooler) [inline app/src/main/kotlin/com/absinthe/libchecker/ui/about/AbsAboutActivityProxy.java:142]: Rewrite all java to Kotlin? With better lambdas.

Comment (Goooler) [reply app/src/main/kotlin/com/absinthe/libchecker/ui/about/AbsAboutActivityProxy.java:142]: #386

Comment (qhy040404) [reply app/src/main/kotlin/com/absinthe/libchecker/ui/about/AbsAboutActivityProxy.java:142]: This is a modified source file whose original file is at https://github.com/PureWriter/about-page/blob/master/library/src/main/java/com/drakeet/about/AbsAboutActivity.java 

Maybe it's not a good idea to rewrite it.

File: app/src/main/kotlin/com/absinthe/libchecker/utils/PackageUtils.kt [hunk 3]
```diff
@@ -464,7 +453,7 @@ object PackageUtils {
     packageInfo.applicationInfo.metaData?.let {
       return it.keySet().asSequence()
         .map { key ->
-          var value = it.get(key).toString()
+          var value = it.getString(key).toString()
```
Comment (Goooler) [inline app/src/main/kotlin/com/absinthe/libchecker/utils/PackageUtils.kt]:
```suggestion
          var value = it.getString(key).orEmpty()
```

**Review by Goooler — APPROVED** [review]

File: app/src/main/kotlin/com/absinthe/libchecker/utils/PackageUtils.kt (hunk 3 above)
Comment (qhy040404) [reply app/src/main/kotlin/com/absinthe/libchecker/utils/PackageUtils.kt]: `get()` returns value or null. `null.toString()` correspond to `"null"`

Comment (Goooler) [reply app/src/main/kotlin/com/absinthe/libchecker/utils/PackageUtils.kt]: Seems no need to be `"null"`.

```kotlin
    if (value.isNotBlank() && value.isDigitsOnly() && value.toLongOrNull() != null) {
      id = value.toLong()
    }
```

**Comment by Goooler** [comment]
And you can rename the title to "Code cleanups" to be more idiomatic.

**Pull request merged**